health-services/ExerciseSampleCompose/app/src/main/java/com/example/exercisesamplecompose/service/ExerciseService.kt

```diff
@@ -197,11 +199,16 @@ class ExerciseService : LifecycleService() {
 
             exerciseNotificationManager.createNotificationChannel()
             val serviceState = exerciseServiceMonitor.exerciseServiceState.value
-            startForeground(
+            ServiceCompat.startForeground(
+                this,
                 ExerciseNotificationManager.NOTIFICATION_ID,
                 exerciseNotificationManager.buildNotification(
-                    serviceState.activeDurationCheckpoint?.activeDuration ?: Duration.ZERO
-                )
+                    serviceState.activeDurationCheckpoint?.activeDuration ?:
+                    Duration.ZERO
+                ),
+                ServiceInfo.FOREGROUND_SERVICE_TYPE_LOCATION or if
```

Comment:
Is this required for this change? If so probably worth a comment.

Or relates to https://github.com/android/health-samples/pull/287/files?

Reply:
Yeah, it relates to #287. For some reason, it included the changes again? Either for good measure, and/or I have a lot to improve on with versioning. 

Added a comment while I'm here

Reply:
I didn't see that change in your PR

Reply:
I think it was an experiment -- Added a comment. It runs without it, as they aren't required values. Is it more readable/developer friendly to have it in a separate commit from this one?